ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml

```diff
@@ -4,46 +4,52 @@
 ##########
 - name: Test if Java 8 is already installed
   win_stat:
-    path: 'C:\Program Files\Java\jdk8u172-b11'
+    path: 'C:\openjdk\jdk8u\bin'
```

Comment:
I would possibly just go with jdk8 to be consistent (the other bootjdk dirs don't seem to use the `u`)

Reply:
I changed all instances of `jdk8u` to `jdk8` and tested it to make sure it works